ReplicationAcceptance/ReplicationAcceptance/CloudantReplicationBase.m

```diff
@@ -12,15 +12,20 @@
 #import "CloudantSyncEncryption.h"
 
 #import "ReplicationSettings.h"
+#import "CDTLogging.h"
 
 #import <UNIRest.h>
 
+
+
 @implementation CloudantReplicationBase
 
+
 - (void)setUp
 {
     [super setUp];
     // Put setup code here; it will be run once, before the first test case.
+    CDTChangeLogLevel(CDTREPLICATION_LOG_CONTEXT, DDLogLevelAll);
```

Comment:
This is useful, if we don't already, this should be in all test base classes and controllable via environment variables or similar.